### ISSUE TITLE
Text field on Android spews lots of messages to the console

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#10730]
```
W/IInputConnectionWrapper( 2043): getTextAfterCursor on inactive InputConnection
W/IInputConnectionWrapper( 2043): getSelectedText on inactive InputConnection
W/IInputConnectionWrapper( 2043): endBatchEdit on inactive InputConnection
E/SpannableStringBuilder( 2043): SPAN_EXCLUSIVE_EXCLUSIVE spans cannot have a zero length
E/SpannableStringBuilder( 2043): SPAN_EXCLUSIVE_EXCLUSIVE spans cannot have a zero length
W/IInputConnectionWrapper( 2043): beginBatchEdit on inactive InputConnection
W/IInputConnectionWrapper( 2043): getTextBeforeCursor on inactive InputConnection
W/IInputConnectionWrapper( 2043): getTextAfterCursor on inactive InputConnection
W/IInputConnectionWrapper( 2043): getSelectedText on inactive InputConnection
W/IInputConnectionWrapper( 2043): endBatchEdit on inactive InputConnection
W/IInputConnectionWrapper( 2043): finishComposingText on inactive InputConnection
W/IInputConnectionWrapper( 2043): beginBatchEdit on inactive InputConnection
W/IInputConnectionWrapper( 2043): getTextBeforeCursor on inactive InputConnection
W/IInputConnectionWrapper( 2043): getTextAfterCursor on inactive InputConnection
W/IInputConnectionWrapper( 2043): getSelectedText on inactive InputConnection
W/IInputConnectionWrapper( 2043): endBatchEdit on inactive InputConnection
W/IInputConnectionWrapper( 2043): beginBatchEdit on inactive InputConnection
W/IInputConnectionWrapper( 2043): getTextBeforeCursor on inactive InputConnection
W/IInputConnectionWrapper( 2043): getTextAfterCursor on inactive InputConnection
```

cc @jason-simmons

**Comment by jason-simmons** [comment]
This should be fixed now.  Are you using an engine with https://github.com/flutter/engine/commit/834fb9638e6087418ce9fb7a690c74f8cbe4c134?

**Comment by Hixie** [comment]
Oh cool, let me try again. I haven't pulled today.

**Comment by Hixie** [comment]
Yup, I still get it. Steps to reproduce: Gallery, Text Fields, put some text in the Life story text field that spans two lines, tap the Name field, long-press a word in the Life story field.
